Add a note to PrivateLink endpoint service of dedicated cluster (#13188)
pingcap/docs: tidb-cloud/set-up-private-endpoint-connections.md

### Step 3. Create an AWS interface endpoint

> **Note:**
>

> > The endpoint service of each dedicated cluster will be automatically created for newly created after March 28.

Reviewer: Adding year here to make it more precise

Author: done

> > The endpoint service of each dedicated cluster will be automatically created for newly created after March 28, 2023.

Reviewer 2: ```suggestion
> For each Dedicated Tier cluster created after March 28, 2023, the corresponding endpoint service is automatically created 3 to 4 minutes after the cluster creation.
```

> TiDB Cloud begins creating an endpoint service, which takes 3 to 4 minutes.
> 
> When the endpoint service is created, take a note of your endpoint service name from the command in the lower area of the console.

Reviewer 2: ```suggestion
If you see the `Endpoint Service Ready` message, take note of your endpoint service name from the command in the lower area of the console for later use. Otherwise, wait for 3 to 4 minutes to let TiDB Cloud create an endpoint service for your cluster.
```

### Step 3. Create an AWS interface endpoint

> **Note:**
>
> For each Dedicated Tier cluster created after March 28, 2023, the corresponding endpoint service is automatically created 3 to 4 minutes after the cluster creation.

> If you see the `Endpoint Service Ready` message, take note of your endpoint service name from the command in the lower area of the console for later use. Otherwise, wait for 3 to 4 minutes to let TiDB Cloud create an endpoint service for your cluster.

Reviewer 2: ```suggestion
If you see the `Endpoint Service Ready` message, take note of your endpoint service name from the command in the lower area of the console for later use. Otherwise, wait 3 to 4 minutes to let TiDB Cloud create an endpoint service for your cluster.
```

The possible statuses of a private endpoint are explained as follows:

> - **Not Configured**: The endpoint server is just created but have not yet created a private endpoint.

Reviewer: ```suggestion
- **Not Configured**: The endpoint service is just created but have not yet created a private endpoint.
```

> - **Not Configured**: The endpoint service is just created but have not yet created a private endpoint.

Reviewer 2: ```suggestion
- **Not Configured**: The endpoint service is created but the private endpoint is not created yet.
```
